src/widget.js

```diff
@@ -231,6 +231,51 @@ function pageVisited(change) {
 	});
 }
 
+function initialize(params) {
+	for (const method in params) {
+		switch(method) {
+			case 'customFied':
```

Comment:
We could add a `setCustomFields` which expects for an array of custom fields and also add it to the `initialized` function.
What to you think?

Reply:
cool :+1: